test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/Execution/BaseRunTestsTests.cs

```diff
@@ -799,6 +866,18 @@ private class TestableBaseRunTests : BaseRunTests
             {
             }
 
+            public TestableBaseRunTests(
```

Comment:
Can we reuse above .ctor?

Reply:
Was not sure how it will disturb other test, & doing it now will mean I can't get devdiv PR out today.